### PR TITLE
Syncronize my work with the new repository.  

### PR DESCRIPTION
**Pull request by Matthewrbowker**
Bringing what I have up to date, though it's not complete.

### CODE REVIEW

**Comment by Matthewrbowker** [comment]
Travis is blocking this PR by #419 now.

File: tests/includes/PdoDatabaseTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * All code in this file is released into the public domain by the ACC        *
+ * Development Team. Please see team.json for a list of contributors.         *
+ ******************************************************************************/
+
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 12/24/16
+ * Time: 00:52
+ */
```
Comment (stwalkerster) [inline tests/includes/PdoDatabaseTest.php]: This header is unnecessary.

If you want to disable it, just empty the PHP File Header template:
image

File: tests/includes/PdoDatabaseTest.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class PdoDatabaseTest extends \PHPUnit_Framework_TestCase
+{
+	/** @var PdoDatabase */
+	private $pdb;
+
+	public function setUp()
+	{
+        $this->markTestIncomplete("Mocking issues.");
+		$this->pdb = $this->getMock("PdoDatabase", array('get_database', 'getDatabaseConnection','get_arguments'));
```
Comment (stwalkerster) [inline tests/includes/PdoDatabaseTest.php]: 4 spaces please, not tabs.

image

File: tests/includes/PdoDatabaseTest.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ * Date: 12/24/16
+ * Time: 00:52
+ */
+
+namespace Waca\Tests;
+
+use Waca\Exceptions\EnvironmentException;
+use Waca\PdoDatabase;
+
+class PdoDatabaseTest extends \PHPUnit_Framework_TestCase
```
Comment (stwalkerster) [inline tests/includes/PdoDatabaseTest.php]: I'd be tempted to add a `use` statement for this

File: tests/includes/DataObjectTest.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<?php
+/******************************************************************************
+ * Wikipedia Account Creation Assistance tool                                 *
+ *                                                                            *
+ * All code in this file is released into the public domain by the ACC        *
+ * Development Team. Please see team.json for a list of contributors.         *
+ ******************************************************************************/
+
+
```
Comment (stwalkerster) [inline tests/includes/DataObjectTest.php]: `namespace Waca\Tests;`

File: tests/includes/Helpers/DebugHelperTest.php [hunk 5]
```diff
@@ -19,7 +19,7 @@ class DebugHelperTest extends PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->dbh = $this->getMock(DebugHelper::class, array("get_debug_backtrace"));
+        $this->dbh = $this->getMockBuilder(DebugHelper::class)->setMethods(["get_debug_backtrace"])->getMock(); //(DebugHelper::class, array("get_debug_backtrace"));
```
Comment (stwalkerster) [inline tests/includes/Helpers/DebugHelperTest.php]: Don't comment out old code, just delete it. We have git if we want to look at old code 😄

**Pull request closed without merging**